### ISSUE TITLE
Temprary Disable(mask/unmask) the dietpi controlled application

### ISSUE THREAD

**Issue by id4vip** [ravenclaw900/DietPi-Dashboard#46]
I just want to request please add these four commands for all dietpi installed software in dietpi-dashboard.

systemctl stop proftpd

systemctl start proftpd

systemctl mask proftpd

systemctl unmask proftpd

as in my system I have installed proftpd, dropbear, samba share etc. Mostly these item I do not need. As if in dietpi-dashboard these commands added then it is very handy to stop ideal programs. I know stop command already showed in dietpi-dashboard, I want to request please add mask/unmask option also. As even if I stop that application through dietpi-dashboard after reboot it again online only mask/unmask command temporary suspend restarting of application after reboot.

Thanks

**Comment by ravenclaw900** [comment]
Hello,
To be clear, along with stop/start, you also want mask/unmask shown in the Services page?

**Comment by id4vip** [comment]
yes, exactly that thing. with that it is very handy to temporary suspend all unneeded applications. & whenever that application needed just switch on that through dietpi-dashboard. it will help reduce hardware resources in long run as well as less active applications reduce system hacking exposure.

**Comment by MichaIng** [comment]
Good idea. I want to get rid of the `dietpi-services` startup at boot but have all services "enabled" instead and regularly started by systemd. This just requires a lot of "After=" additions to the systemd units to assure backends/databases are started before frontend applications. When this is done, enable/disable switch to not start a service at boot would be great as well. That way it it can be still started manually without the need to first unmask it.
